ocean_provider/file_types/file_types.py

```diff
@@ -34,7 +35,8 @@ def validate_dict(self) -> Tuple[bool, Any]:
 
         if self.method not in ["get", "post"]:
             return False, f"Unsafe method {self.method}."
-
+        if not is_safe_url(self.url):
```

Comment:
can you please add a line before and after this if+return, to make the code more readable?